examples/imdb.py

```diff
@@ -288,7 +290,8 @@ def main():
 
     for epoch in range(1, args.epochs + 1):
         train(args, model, train_loader, optimizer, epoch)
-        evaluate(args, model, test_loader)
+        mean_accuracy = evaluate(args, model, test_loader)
+        torch.save(mean_accuracy, "run_results_imdb_classification.pt")
```

Comment:
nit: probably no point in saving it after each epoch?

Reply:
thanks for spotting!